tests/test_issue_3134_ghostty_sudo_alias_zsh_bootstrap.py

```
@@ -0,0 +1,99 @@
+        for filename in (".zprofile", ".zshrc"):
+            (orig / filename).write_text("", encoding="utf-8")
+
+        # Use the real Ghostty-style `sudo()` definition here: with
+        # `alias sudo='sudo '`, zsh can choke while parsing this exact form.
+        (bundled / "ghostty-integration.zsh").write_text(
+            "sudo() { :; }\n"
+            'print -r -- "loaded" >> "$CMUX_TEST_OUT"\n',
+            encoding="utf-8",
+        )
+
+        env = dict(os.environ)
```

Ruff (0.15.12)

[error] Undefined name `os`

(F821)